### ISSUE TITLE
Pulling an environment into the frontend application fails silently

### ISSUE THREAD

**Issue by sergiuionescu** [aws-amplify/amplify-cli#7539]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm i -g @aws-amplify/cli@5.0.0

### If applicable, what version of Node.js are you using?

14

### Amplify CLI Version

5.0.0

### What operating system are you using?

Mac

### Amplify Categories

auth

### Amplify Commands

pull

### Describe the bug

Cannot fetch `aws-exports.js` for a frontend app, the process ends without any error after the Admin UI Authorization step.

Can you please advise how i could debug this issue further?

Thanks.

### Expected behavior

The process continues, you follow through the local configuration in the cli and obtain the `aws-exports.js` file.

### Reproduction steps

I am trying to pull an existing backend environment into a react app with:

```
amplify pull --appId <appId> --envName dev
```
I can follow the process  through the browser authorization:

```
Opening link: https://eu-central-1.[appAuthorizeUrl]
✔ Successfully received Amplify Admin tokens.
Amplify AppID found: <appId>. Amplify App name is: <appName>
Backend environment dev found in Amplify Console app: <appName>
```

Actual behavior: The process ends without any error and a 0 exist code, the `aws-exports.js` file is not created.

### Log output

No errors are logged, only info with identical output to another machine where this is working properly.

**Comment by josefaidt** [comment]
Hey @sergiuionescu :wave: were you prompted to answer the following questions after authenticating?

```shell
> amplify pull --appId <appId> --envName dev
Opening link: https://us-east-1.admin.amplifyapp.com/admin/<appId>/dev/verify/
✔ Successfully received Amplify Admin tokens.
Amplify AppID found: <appId>. Amplify App name is: <appName>
Backend environment dev found in Amplify Console app: <appName>
? Choose your default editor: Visual Studio Code
? Choose the type of app that you're building javascript
Please tell us about your project
? What javascript framework are you using none
? Source Directory Path:  src
? Distribution Directory Path: dist
? Build Command:  npm run-script build
? Start Command: npm run-script start
? Do you plan on modifying this backend? No
```

**Comment by sergiuionescu** [comment]
@josefaidt No, that's just the thing the prompt for the editor never happened and the exit code was 0 `echo $?`.

The issue was resolved after updating npm from 7.5.4 and node from v14.0.0 to v14.17.0 .

Sorry, not much details here, but updating npm, node and reinstalling the cli : `npm i -g @aws-amplify/cli@5.0.0` seems like a good solution.

**Comment by josefaidt** [comment]
@sergiuionescu thanks for the information, I'll take note of this limitation for future reference. Given the issue is related to node/npm, closing issue. If you run into something like this again please feel free to comment back and we can re-open.
